TESTING.md

```diff
@@ -27,8 +27,8 @@ In general, we recommend four tiers of tests:
 
 
 # Requirements
-* Install the latest NodeJS, [NPM](https://www.npmjs.com/get-npm) and [Yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
-    * `nvm install v18.19.0`
+* Install the specified NodeJS, [NPM](https://www.npmjs.com/get-npm) and [Yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
```

Comment:
Why would we require a "specific" version of Node.js for testing?

Reply:
If you install the latest NodeJS version it won't build.

Reply:
nit: we may want to advise users to simply run `nvm install` so that they always get the version defined in our `.nvmrc` in the case these docs get outdated.

Not needed for this change though.